.circleci/config.yml

```diff
@@ -94,6 +94,8 @@ executors:
       SUBSCRIPTIONS_PAYPAL_ENABLED: true
       PAYPAL_SANDBOX: true
       FXA_MX_RECORD_EXCLUSIONS: restmail.dev.lcip.org
+      # Needed for react conversion playwright tests
+      REACT_CONVERSION_SIMPLE_ROUTES: true
```

Comment:
Is this really all we needed here?! Lol.

Is it possible to launch a specific test suite with these flags on? I'm trying to picture how we'll temporarily (until we sunset content-server) want all of our functional tests to test our React pages with that param. Also something to know, a lot of the selectors in our functional tests are IDs that we don't necessarily have in the React conversions and I'm hoping we can adjust those to either match text selectors how we have in our unit tests or change them to `data-testid` and add those IDs onto the elements in question.

Reply:
I think this was the most straightforward way? I looked into adding a circleci workflow but that was meh. To test the different flows, I thought that we would have a different test file just for react conversion (similar to custom tests when migrating settings)

If we have a naming convection for the tests (prefix `react conversion`), we should be able to do `yarn test-local --grep="react conversion"`.

Reply:
Heheh, I'm just surprised since I thought it was going to be more complex 😆

Do you mind adding all the other env vars here too? I don't think that'll cause any issues since you have to be in the experiment or have that `showReactApp` param present. I think there's definitely going to be flows/tests where we'll want to append the experiment params instead, maybe we should set that up here too?

> To test the different flows, I thought that we would have a different test file just for react conversion (similar to custom tests when migrating settings)

Hmm I suppose we will essentially be duplicating all of our tests that touch content-server temporarily then. In thinking about launching our pages to production with confidence, I think that's what we'll want to do if we can't launch, hopefully they can be organized in such a way in the package that it's easy to tell which is which. Duplicating tests makes updating selectors easier too since those will differ. We can always add a cleanup task to remove the React tests in the "sunset content-server" epic, so 👍